concepticondata/conceptlists/Lastra-1986-381.tsv

```diff
@@ -0,0 +1,382 @@
+Lastra-1986-381-89	89	guajolote (si además distiguen por tamaño y sexo, anotar enfrente)	Guajolote (if they also distribute by size and sex, write down in front)	2386	TURKEY
+Lastra-1986-381-90	90	gallina	hen	1514	HEN
+Lastra-1986-381-91	91	gavilán	hawk	1865	BUZZARD
+Lastra-1986-381-92	92	zopilote	buzzard		
+Lastra-1986-381-93	93	tecolotes	Tecolotes		
+Lastra-1986-381-94	94	pájaro carpintero (o cherreque)	woodpecker		
+Lastra-1986-381-95	95	iguana	Iguana	1578	IGUANA
+Lastra-1986-381-96	96	lagartija	lizard	632	LIZARD
+Lastra-1986-381-97	97	conejos	rabbits		
+Lastra-1986-381-98	98	venados	deers		
+Lastra-1986-381-99	99	cerdo	pig	1337	PIG
+Lastra-1986-381-100	100	mi borrego	My sheep		
+Lastra-1986-381-101	101	armadillo	armadillo	1885	ARMADILLO
+Lastra-1986-381-102	102	ardilla	squirrel	1628	SQUIRREL
+Lastra-1986-381-103	103	mapache	raccoon	828	RACCOON
+Lastra-1986-381-104	104	tlacuache	opossum	309	OPOSSUM
+Lastra-1986-381-105	105	zorillo	skunk	3888	SKUNK
+Lastra-1986-381-106	106	muerciélago	bat	1793	BAT
+Lastra-1986-381-107	107	mariposa	butterfly	1791	BUTTERFLY
+Lastra-1986-381-108	108	pescado	fish	227	FISH
+Lastra-1986-381-109	109	ratón	mouse	1491	MOUSE
+Lastra-1986-381-110	110	rata	rat	1490	RAT
+Lastra-1986-381-111	111	perro	dog	2009	DOG
+Lastra-1986-381-112	112	mi perro	my dog		
+Lastra-1986-381-113	113	rana	frog	503	FROG
+Lastra-1986-381-114	114	sapo	toad	894	TOAD
+Lastra-1986-381-115	115	caracol	snail	1543	SNAIL
+Lastra-1986-381-116	116	gusano	worm	1219	WORM
+Lastra-1986-381-117	117	alacrán	scorpion	1538	SCORPION
+Lastra-1986-381-118	118	garrapata	tick	1527	TICK
+Lastra-1986-381-119	119	liendre	nit	267	NIT
+Lastra-1986-381-120	120	piojo (diferenciar del piojo blanco)	Piojo (differentiate from white louse)	1392	LOUSE
+Lastra-1986-381-121	121	pulga	flea	232	FLEA
+Lastra-1986-381-122	122	pulgas	fleas		
+Lastra-1986-381-123	123	cucaracha	cockroach	1575	COCKROACH
+Lastra-1986-381-124	124	hormiga negra	black garden ant		
+Lastra-1986-381-125	125	hromiga arriera	atta ant		
+Lastra-1986-381-126	126	hormiguero	anthill	1873	ANTHILL
+Lastra-1986-381-127	127	telaraña	spiderweb	1065	SPIDER WEB
+Lastra-1986-381-128	128	semilla (de calabaza)	seed (from pumpkin)		
+Lastra-1986-381-129	129	corteza	bark	1204	BARK
+Lastra-1986-381-130	130	raíz	root	670	ROOT
+Lastra-1986-381-131	131	árbol	tree	906	TREE
+Lastra-1986-381-132	132	hoja de (árbol)	tree leaf	628	LEAF
+Lastra-1986-381-133	133	horqueta	pitchfork	337	PITCHFORK
+Lastra-1986-381-134	134	nopal	prickly pear		
+Lastra-1986-381-135	135	plátano (especificar la clase)	banana (specify the class)	868	BANANA
+Lastra-1986-381-136	136	guayaba	guava	3981	GUAVA
+Lastra-1986-381-137	137	guayabas	Guayabas		
+Lastra-1986-381-138	138	frijol	bean	832	BEAN
+Lastra-1986-381-139	139	chayote	chayote		
+Lastra-1986-381-140	140	maguey	agave		
```

Comment:
A concept for `AGAVE` would be good, we've got quite a number of unmapped instances now.

Reply:
I also thought of this while checking the list. I'll open an issue about this.